pkgs/repo_query/lib/labels.dart

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:math';
+
+import 'package:github/github.dart';
+
+import 'src/common.dart';
+
+class LabelsCommand extends ReportCommand {
+  LabelsCommand()
+      : super('labels',
+            'Report on the various labels in use by dart-lang repos.');
+
+  @override
+  Future<int> run() async {
+    var repos = await getReposForOrg('dart-lang');
+
+    var results = <Repository, List<IssueLabel>>{};
+
+    for (var repo in repos) {
+      var labels =
+          await reportRunner.github.issues.listLabels(repo.slug()).toList();
+
+      results[repo] = labels;
+
+      print('${repo.slug} has ${results[repo]!.length} labels '
+          '(${repo.openIssuesCount} issues, ${repo.stargazersCount} stars).');
+    }
+
+    print('');
+
+    // calculate label usage
+    var labels = <String, _LabelInfo>{};
+
+    for (var entry in results.entries) {
+      var repo = entry.key;
+      if (repo.openIssuesCount < 30) continue;
+
+      for (var label in entry.value) {
+        var labelInfo =
+            labels.putIfAbsent(label.name, () => _LabelInfo(label.name));
+
+        labelInfo.repos.add(repo.name);
+        labelInfo.weight += log(repo.openIssuesCount);
```

Comment:
Maybe extract the two occurrences of `repo.openIssuesCount` to a local variable, to make the previous check for `openIssuesCount<30` more clear?

Reply:
done! (and, added a comment about the issue count cutoff)